examples/image_input/.ipynb_checkpoints/image_input_tutorial-checkpoint.ipynb

```diff
@@ -0,0 +1,112 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "03269354-1f22-4ef9-9eb9-b1825bc95724",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import Image, display, Audio, Markdown\n",
+    "from image_input import ImageInput\n",
+    "\n",
+    "List_image_path = [\"/path/image1.png\"]\n",
+    "\n",
+    "# Preview image for context\n",
+    "display(Image(List_image_path[0]))\n",
+    "\n",
+    "# Open the image file and encode it as a base64 string\n",
+    "\n",
+    "image_input = ImageInput(List_image_path)\n",
+    "image_base64 = image_input.encode()\n",
+    "image_extension = image_input.extension()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4b58fd68-7dbf-49eb-9b9e-03a93dd70a3e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import requests\n",
+    "from dotenv import load_dotenv\n",
+    "\n",
+    "load_dotenv()\n",
+    "\n",
+    "# Configuration\n",
+    "URL = os.getenv('HOST')+':'+os.getenv('PORT')+'/'+os.getenv('API_ENDPOINT_PREFIX')\n",
+    "ENDPOINT = \"/v1/chat/completions\"\n",
+    "\n",
+    "# data for the request\n",
+    "data = f'''{{\n",
+    "    \"messages\": [\n",
+    "      {{\n",
+    "        \"role\": \"user\",\n",
+    "        \"content\": [\n",
+    "            {{\"type\": \"text\", \"text\": \"Summarize all the information included in this image.\"}},\n",
+    "            {{\n",
+    "                \"type\": \"image_url\",\n",
+    "                \"image_url\": {{\n",
+    "                    \"url\": \"data:image/{image_extension[0]};base64,{image_base64[0]}\"\n",
```

Comment:
Put an example with several images otherwise, the class shouldn't accept a list of images but one single image